SWHttpTrafficRecorder.m

```diff
@@ -75,11 +75,12 @@ - (BOOL)startRecordingAtPath:(NSString *)recordingPath forSessionConfiguration:(
             NSFileManager *fileManager = [NSFileManager defaultManager];
             if(![fileManager fileExistsAtPath:recordingPath]){
                 NSError *bError = nil;
-                [fileManager createDirectoryAtPath:recordingPath withIntermediateDirectories:YES attributes:nil error:&bError];
-                if(bError && error){
-                    *error = [NSError errorWithDomain:SWHttpTrafficRecorderErrorDomain code:SWHttpTrafficRecorderErrorPathFailedToCreate userInfo:@{NSLocalizedDescriptionKey: [NSString stringWithFormat:@"Path '%@' does not exist and error while creating it.", recordingPath]}];
+                if(![fileManager createDirectoryAtPath:recordingPath withIntermediateDirectories:YES attributes:nil error:&bError]){
```

Comment:
Good catch. Shall we consider the case when bError would not be nil? Currently it is totally ignored.